src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/components/Results.tsx

```diff
@@ -49,7 +50,7 @@ export const Results: React.FC<Props> = ({
   const [errorMessage, setErrorMessage] = useState<string>('');
   const {t} = useTranslation();
 
-  const availableTripPatterns = useAvailableTripPatterns(tripPatterns);
+  const now = useNow(2500);
```

Comment:
Since we now need to evaluate the time in the `Results` component, this will introduce quite a bit of rerenders of what might be a long list. I think it is worth to handle this better, as we have seen some performance issues in the app lately.

Possible options are:
- Use `React.memo` so only the result items that actually needs to rerender are rerendered.
- Skip using `useNow`, which makes the booking time only evaluated at search time. Which might be good enough?